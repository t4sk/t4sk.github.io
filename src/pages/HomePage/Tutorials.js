import React from "react"
import PropTypes from "prop-types"
import styles from "./Tutorials.module.css"
import TutorialCard from "./TutorialCard"

import learnSolidityCardLogo from "../../static/learn-solidity-card.svg"

const COURSES = [
  {
    title: "Learn Solidity",
    logo: learnSolidityCardLogo,
    href: "/tutorials/learn-solidity",
  },
]

export function Tutorials(props) {
  return (
    <div className={styles.component}>
      {COURSES.map(course => (
        <TutorialCard key={course.title} {...course} />
      ))}
    </div>
  )
}

Tutorials.propTypes = {}

export default Tutorials
