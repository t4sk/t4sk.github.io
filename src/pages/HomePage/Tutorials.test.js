import React from "react"
import { shallow } from "enzyme"
import { Tutorials } from "./Tutorials"

const props = {}

beforeEach(() => {})

test("it renders", () => {
  const component = shallow(<Tutorials {...props} />)

  expect(component).toMatchSnapshot()
})
