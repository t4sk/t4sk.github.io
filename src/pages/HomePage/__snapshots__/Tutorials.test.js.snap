// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  className="component"
>
  <div
    className="card"
    key="Learn Solidity"
    onClick={[Function]}
  >
    <TutorialCard
      logo="solidity.svg"
      onClick={[Function]}
      path="/tutorials/learn-solidity"
      title="Learn Solidity"
    />
  </div>
  <div
    className="card"
    key="Build a Multi-Sig Wallet in Solidity"
    onClick={[Function]}
  >
    <TutorialCard
      logo="solidity-app.svg"
      onClick={[Function]}
      path="/tutorials/solidity-multi-sig-wallet"
      title="Build a Multi-Sig Wallet in Solidity"
    />
  </div>
</div>
`;
